lib/temporal/testing/local_workflow_context.rb

```diff
@@ -57,7 +57,7 @@ def execute_activity(activity_class, *input, **args)
           attempt: 1,
           workflow_run_id: run_id,
           workflow_id: workflow_id,
-          workflow_name: nil, # not yet used, but will be in the future
+          workflow_name: self.metadata.name,
```

Comment:
`spec/integration/local_hello_world_workflow_spec.rb` crashes without `self.` - I think because line 52 hides self.metadata with the uninitialized local variable.